client/accounting.go

```diff
@@ -35,9 +36,12 @@ func (x *PrmBalanceGet) SetAccount(id user.ID) {
 //   - [ErrMissingAccount]
 func (c *Client) BalanceGet(ctx context.Context, prm PrmBalanceGet) (accounting.Decimal, error) {
 	var err error
-	defer func() {
-		c.sendStatistic(stat.MethodBalanceGet, err)()
-	}()
+	if c.prm.statisticCallback != nil {
```

Comment:
Likely you can just `defer c.sendStatistic(...)()` and handle `statisticCallback` and `startTime` internally.

Reply:
if `if` is inevitable, then it is better to make a conditional `defer` imo

Reply:
but why? you turned out the func that is commonly used and made more lines, no? i think it was planned as `defer c.sendStatistic(...)()` but smth went wrong, why make things harder?

Reply:
Not doing a `defer` is obviously more efficient, but for these (network) operations the difference is so negligible that I'd rather opt for simpler code.

Reply:
> at least i would try to inline time calculation: `time.Since(time.Now())`

hm, how would it be?

Reply:
it can be an arg of the deferred func

Reply:
done